packages/components/src/styles/chip/chip.stories.ts

```diff
@@ -1,64 +1,48 @@
 import '../../solid-components';
+import { html } from 'lit';
+import { storybookDefaults, storybookHelpers } from '../../../scripts/storybook/helper';
 
-import { FlagSamples } from '../flag/flag.stories';
-import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-
-const { argTypes, parameters } = storybookDefaults('sd-chip');
+const { argTypes } = storybookDefaults('sd-chip');
 const { overrideArgs } = storybookHelpers('sd-chip');
-const { generateTemplate } = storybookTemplate('sd-chip');
 
 /**
- * A small, non-interactive label the represents a status, property or meta-data.
+ * A small, non-interactive label that represents a status, property or meta-data.
+ *
+ *  **Related templates**:
+ * - [Chip](?path=/docs/templates-chip--docs)
+ * - [Teaser with Chip and Flag](?path=/docs/templates-teaser-with-chip-and-flag--docs)
  *
- * <b>Variants</b><br>
- * <li>--primary-200 is the default variant</li>
- * <li>--primary-500</li>
- * <li>--primary-300</li>
- * <li>--white</li>
  */
 
 export default {
   title: 'Styles/sd-chip',
   component: 'sd-chip',
-  parameters: {
-    ...parameters,
-    backgrounds: {
-      default: 'neutral-200'
-    },
-    design: {
-      type: 'figma',
-      url: 'https://www.figma.com/file/IUiRoK2jiW8ydM77uiY2RX/Chip?type=design&mode=design&t=cioeESUO1sJ6UIu8-0'
-    }
-  },
+  tags: ['!dev'],
   args: overrideArgs({ type: 'slot', name: 'default', value: 'Lorem Ipsum' }),
   argTypes
 };
 
 export const Default = {
-  render: (args: any) => {
-    return generateTemplate({
-      options: { templateContent: '<span class="%CLASSES%">%SLOT%</span>' },
-      args
-    });
-  }
+  name: 'Default',
+  render: () =>
+    html` <div class="flex gap-12 bg-neutral-100 p-8">
+      <div class="sd-chip sd-chip--primary-200">primary-200</div>
+    </div>`
 };
-
+/**
+ * Use the `classes` variant for alternative appearances:
+ * - `--primary-200` (default): use the class `sd-chip--primary-200`
+ * - `--primary-300`: use the class `sd-chip--primary-300`
+ * - `--primary-500`: use the class `sd-chip--primary-500`
+ * - `--white`: use the class `sd-chip--white`
```

Comment:
```suggestion
 * Use `sd-chip` modifiers for alternative appearances:
 * - `--primary-200` (default)
 * - `--primary-300`
 * - `--primary-500`
 * - `--white`
```

This is not a classes variant.

@MartaPintoTeixeira I think it makes sense to use this here: https://solid-design-system.github.io/solid/feature_sd-chip-docs/?path=/docs/templates-chip--docs

As you really have recommendations, I believe this could be part of the docs. From my opinion you could just replace the docs here and show the one from the template here. What do you think? @MartaPintoTeixeira  @coraliefeil